node/src/blockchain/blockchain_interface.rs

```diff
@@ -1906,20 +2039,6 @@ mod tests {
         assert_eq!(converted_wei, U256::from_dec_str("1000000000").unwrap());
     }
 
-    #[test]
```

Comment:
I understand getting rid of the part of this test that tests the behavior of the `U256` type; but why get rid of the part of it that verifies the value of the `GWEI` constant?

Reply:
Do we still need the conversion? Bert said the amounts are in `wei` now. Regardless I restored the test.